include/rocksdb/db.h

```diff
@@ -294,6 +294,12 @@ class DB {
   // delete it after use.
   //
   // Return OK on success, non-OK on failures.
+  //
+  // WARNING: Secondary databases cannot read shared SST files that have been
+  // truncated in the primary database. To avoid compatibility issues, users
+  // should refrain from using features in the primary database that can cause
+  // truncation, such as setting `bytes_max_delete_chunk > 0` when invoking
```

Comment:
nit: `rate_bytes_per_sec` is another setting that enables slow deletion: https://github.com/facebook/rocksdb/blob/e6972196bca115e841a6b88d361ba945b49e1e5d/file/delete_scheduler.cc#L64

These two both need to be > 0 in order for `ftruncate` to be used for file deletion.

Reply:
Fixed